public/components/custom_panels/home.tsx

```diff
@@ -297,9 +299,10 @@ export const Home = ({
         side={toastRightSide ? 'right' : 'left'}
         toastLifeTimeMs={6000}
       />
+      <Switch>
       <Route
         exact
-        path={renderProps.match.path}
+        path={['/operational_panels/create', '/operational_panels']}
```

Comment:
is 'Create panel' now the also the home?

Reply:
'/operational_panels' is still the home but when used '/operational_panels/create', just the url changes and in the background its still home page